src/index.js

```diff
@@ -1,68 +1,38 @@
 import './style.css';
 
-function Task(description, index) {
-  this.description = description;
-  this.completed = false;
-  this.index = index;
-}
+import display from './display.js';
 
-let tasknumber = 0;
+import changebutton from './changebutton.js';
 
-const arr = [
-  {
-    description: 'example task',
-    completed: false,
-    index: 0,
-  },
-];
-const tasklist = document.querySelector('#dynamiccontainer');
-window.localStorage.setItem('tasklist', JSON.stringify(arr));
+import { list } from './class.js';
 
-const record = () => {
-  let arr = JSON.parse(window.localStorage.getItem('tasklist'));
-  if (arr === null) {
-    arr = [];
-  }
+document.querySelector('#taskenter').addEventListener('click', () => {
   const taskname = document.querySelector('#addTask').value;
-  const task = new Task(taskname, tasknumber);
-  arr.push(task);
-  const tasklist = document.querySelector('#dynamiccontainer');
-  tasklist.innerHTML += `<div class="field">
-    <div class="inputcontainer">
-      
-      <label for=${tasknumber}><input type="checkbox" class="task" id=${tasknumber}>${taskname}</label>
-    </div>
-    <button class="listbutton" id=${tasknumber} type="button"><ion-icon name="ellipsis-vertical"></ion-icon></button>
-  </div>`;
-  tasknumber += 1;
-  window.localStorage.setItem('tasklist', JSON.stringify(arr));
-};
+  list.add(taskname);
+  display();
+});
 
-document.querySelector('#taskenter').addEventListener('click', () => {
-  record();
+document.querySelector('#form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const taskname = document.querySelector('#addTask').value;
+  list.add(taskname);
+  display();
 });
 
 const clearer = () => {
-  const arr = JSON.parse(window.localStorage.getItem('tasklist'));
-  arr.forEach((item) => {
-    if (item.complete === true) {
-      delete(item.index);
-    }
-  });
+  let arr = JSON.parse(window.localStorage.getItem('tasklist'));
+  let index = JSON.parse(window.localStorage.getItem('tasknumber'));
+  arr = [];
+  index = 0;
+  window.localStorage.setItem('tasknumber', JSON.stringify(index));
   window.localStorage.setItem('tasklist', JSON.stringify(arr));
-  const tasklist = document.querySelector('#dynamiccontainer');
-  tasklist.innerHTML = '';
+  display();
 };
 
 document.querySelector('#clear').addEventListener('click', () => {
   clearer();
 });
-
-const changebutton = (id) => {
-  const button = document.querySelector(id);
-  button.innerHTML = '<ion-icon name="trash-outline"></ion-icon>';
-};
-
+/*
 const complete = (boxIndex) => {
```

Comment:
- [ ] **Optional:** Your work is more difficult to read and it is unprofessional to leave unused codes in the codebase. Kindly remove them.